### DISCUSSION TITLE
People: Improvement of the face:new feature

### DISCUSSION THREAD

**Discussion by kingvuke** [photoprism/photoprism#3808] [Ideas]
As a user, I would like to see all new faces (face:new) on one page.
The faces should be sorted by similarity.
I can select a couple of faces, and assign to the person.

See as example https://docs.digikam.org/en/_images/mainwindow_faces_tag_assigned_iconview.webp

**Comment by graciousgrey** [comment]
I'll convert this into a discussion as it duplicates some existing feature requests. Feel free to add a :+1:  to those :)

- https://github.com/photoprism/photoprism/issues/1601
- https://github.com/photoprism/photoprism/issues/1679
- https://github.com/photoprism/photoprism/issues/1595

**Comment by mghoffmann** [comment]
I just started using PhotoPrism a couple of days ago and this is the biggest thing keeping me from completely switching from Google Photos. I can understand its difficulty with recognizing children, but having to tag 6000+ pictures of the same child one at a time is getting tedious 😅

Overall the software is fantastic, especially for the cost. I'm really happy to have an easy way to ingest all of my photos and deduplicate them.
